phpstan.neon

```diff
@@ -0,0 +1,11 @@
+includes:
+  - phpstan-baseline.neon
+
+parameters:
+  level: 9
+  paths:
+    - examples
+    - src
+    - tests
+  excludePaths:
+    - src/Server.php
```

Comment:
I had to exclude this file because otherwise I could not generate the baseline.

This is due to the `Server` class attempting to extend the final `HttpServer` class